src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,116 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.InvalidDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private static final int DEFAULT_AGE = 18;
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "user_login@gmail.com";
+    private static User user;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void getDefaultUser() {
+        user = new User();
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+    }
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_addAdultUser_ok() {
+        User actual = registrationService.register(user);
+        assertEquals(user, actual,"User is added to Storage");
+    }
+
+    @Test
+    void register_userRegistered_notOk() {
+        registrationService.register(user);
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user),
+                "User already exists");
+    }
+
+    @Test
+    void register_nullLogin_noOk() {
+        user.setLogin(null);
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user),
+                "Login can't be null");
+    }
+
+    @Test
+    void register_Age_Ok() {
```

Comment:
don't understand why do you need this test if you have `register_addAdultUser_ok`